README.md

```diff
@@ -4,57 +4,59 @@ ea_restaurant_data_loader is a microservice as part of [ea_restaurant](https://g
-- a database - Phoenix recommends PostgreSQL , but you can pick others or not use a database at all
-- and other optional packages
+* You need to install the [Erlang/OTP](https://www.erlang.org/downloads) and the [Elixir](https://elixir-lang.org/install.html#windows) programming language
 
-In order to get a Phoenix app running and to install any extra dependencies we might need to install Hex.
+* It is necessary to create the files `dev.secrets.exs`, `test.secret.exs`, `prod.secret.exs` inside the config folder to storage all the information that you
+may not share in version control or any place out of workplace. Inside config folder you going to have a `example_secret.exs` file.
 
-Here’s the command to install Hex:
+In order to use secrets inside secret file you can access them using the following code line:
+    * `Application.get_env(:ea_restaurant_data_loader, :secret_key)` # Will return the secret key from .secret.exs file
 
-`mix local.hex`
 
-Once we have Elixir and Erlang, we are ready to install Phoenix application generator:
+### Dependencies
 
-`mix archive.install hex phx_new`
-
-After that the `phx.new` generator is now available to generate new applications. In order to config a database Phoenix applications use another elixir package, called Ecto. Ecto is a toolkit for data mapping and language integrated query for Elixir.
-
-Now we are ready to create our Phoenix application running the following command:
-
-`mix phx.new ea_restaurant_data_loader`
-
-In order to have the phoenix server ready it’s good to fetch and install dependencies running the command:
-
-* `mix deps.get`
+* In order to get the neccesary dependencies for the project you have to run the command:
+    * mix deps.get
 
 Note: It is important before to install the dependencies to install a C compiler to be able to compile `bcrypt_elixir` dependency, follow the following
 steps:
 * Install a recent version of [Visual C++ Build Tools](https://visualstudio.microsoft.com/downloads/) (On the download page, click on tools for Visual Studio and then you will see the link for the build tools for visual studio 2022)
 
 * Run the visual studio tools and install de desktop C++ developer package
 
-* After installing the visual C++ build tools open the `run` command and type the follogin command(make sure that the path and version number are correct):
+* After installing the visual C++ build tools open the `run` command and type the following command(make sure that the path and version number are correct):
     `cmd /K "C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\Common7\Tools\VsDevCmd.bat" amd64`
 
 
 This should open up a command prompt with the necesary enviroment variables set, and from which you will be able to run the commands `mix compile`, `mix deps.compile` and `mix test`. note:cd project path to run the commands above.
```

Comment:
`mix compile` wont work because an env variable is missing. Is `nmake`. That information is missing.
![image](https://github.com/eapg/ea_restaurant_data_loader/assets/46677910/82853ad1-d986-4742-9ec9-42fb87fc84ef)

Reply:
This part was removed.